src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -102,21 +108,20 @@ class ComService(
       probationSearchApiSortBy,
     )
 
```

Comment:
I'd extract the building up of `probationSearchApiSortBy` into an extension function.

This would help normalise the level of abstraction in this function so it's roughly constant across.

Reply:
Also maybe rename `entityProbationSearchResult` -> `teamCaseloadResults` ?

(ProbationSearchResult is naming based on type rather role and entity is generally used to refer to DB entities )

Reply:
Done